README.md

```diff
@@ -223,7 +219,16 @@ The server will respond with JSON object.
 - The applicationId returned can be used to view the candidate profile in Lever at the url: `https://hire.lever.co/search/application/{applicationId}`. Note that only users logged in to Lever will be able to access that page.
 - On error, we'll send the appropriate HTTP error code and a body of `{ok:false, error:<error string>}`.
 
+### WARNING: POST Application Rate limit
+
+**Application create requests are rate limited. Your team will need to properly handle `429` responses if you build a custom job application page.**
```

Comment:
I'd add: "If you are unable to implement this logic, please consider directing to Lever's hosted application form instead of implementing a custom application form. See more below."